src/components/CaseDetails/index.tsx

```diff
@@ -84,6 +88,52 @@ const caseFields = [
 ];
 
 export default function CaseDetails({ caseData }: { caseData: CaseListing }) {
+  const profile = useContext(ProfileContext);
+
+  const Interest = useMemo(() => {
+    if (profile && profile.userId)
+      return profile.profileReady ? (
+        <InterestForm caseData={caseData} />
+      ) : (
+        <>
+          <H3>
+            Please finish submitting your profile before submitting interest.
```

Comment:
sorry if i misled you LOL
`profileReady` only tells us if the user's profile has been loaded, but not whether or not they have been onboarded
however, it can be useful to setup loading states for the future

Reply:
OHHHHH hahah ok will update